apps/examples/static/components/mtable.js

```diff
@@ -3,10 +3,10 @@
     var mtable = { props: ['url', 'filter', 'order', 'editable', 'create', 'deletable', 'render'], data: null, methods: {}};
     
     mtable.data = function() {        
-        var data = {url: this.url,
+        var data = {mtUrl: this.url,
                     busy: false,
-                    filter: this.filter || '',
-                    order: this.order ||  '',
```

Comment:
`filter` and `order`  are used as initial values, so it is better to remove them from props at all and use `$attrs` like
`filter: this.$attrs.filter || ''`

Reply:
$attrs is not compatible with vue 2.2.0 (the version shipped by py4web  currently) because vue introduced it from 2.4.0

https://vuejs.org/v2/api/#vm-attrs